bionic/flow.py

```diff
@@ -462,6 +485,8 @@ def assign(
             Description of the new entity.
         persist: Boolean, optional
             Whether this entity's values should be cached persistently.
+        memoize: Boolean, optional
```

Comment:
This change is to make it more in line with how persist configuration works. It also sets up the configuration for my next PR, where this will be the override value, and we will have a default value for memoization.

I'll add the docstrings for sure.

Reply:
But even after your next PR, will it ever be meaningful for the user to change this value? All fixed values are unavoidably stored in memory, regardless of whether memoization is enabled, right? I don't understand why a user would ever want to set this to False.

Reply:
Also, since we're on the topic: this docstring doesn't make it super clear why a user would want `persist` to be True for a fixed entity.

Reply:
Yeah, it's true that all fixed values will be stored in memory inside the providers. I didn't think of that. I agree with you on this now, we don't need to have this toggle for fixed values and we can always cache them.